wip/On_API_Updates.md

```diff
@@ -99,12 +99,12 @@ random arrays in the range `[80, 120[` for entering and leaving).</div>
 
 I know this example is a little contrived, but bear with me. Suppose we use this service
 for a while before it breaks, like our real-world application did. The stack traces tell
-you there is something wrong with the format that `get_facecolor` returns—they will tell
-you they expected a string, but you gave it a `numpy.ndarray` by trying to call the `find`
-method, which is defined for strings, and failing. You never looked at the color format
+you there is something wrong with the format returned by `get_facecolor`—they tell
+you they expected a string, but you gave it a `numpy.ndarray`.You try to call the `find`
```

Comment:
That's not quite true. The `find` function is called internally, not by us. It's the reason the code fails in the first place. 

I suggest changing it to `It tries to call [...] and fails.`. Is that okay?